ipydatagrid/_version.py

```diff
@@ -1,4 +1,5 @@
 # Copyright (c) Bloomberg.
 # Distributed under the terms of the Modified BSD License.
 
-__version__ = "1.1.16"
+__version__ = "1.1.17"
+__frontend_version__ = "^1.1"
```

Comment:
Keeping them in sync should not really be requirement. Ideally, we would keep a separate version for the model schema. Hypothetically, ipydatagrid 2.x could work with the frontend version 1.x if we don't touch the model schema and only break the user Python API. 
We would version the **model schema** separately from the **Python package** and the **JavaScript package**.
But I'd prefer not maintaining 3 versions on the same repository...

Loosening the requirement on the patch version will allow users to upgrade to ipydatagrid 1.1.17 without refreshing their browsers and it will *just* work as long as they have the 1.1.x frontend version. If we set ourselves to the rule that we don't touch the model schema on patch releases, that works.

In any case, I can revert this change if you really think we should keep it in sync. I don't have a strong feeling about it.

Reply:
And thanks a lot for the review!

Reply:
We co-maintain the frontend and the backend in the same repository and is not intended to mix the frontend and backend versions. I'd prefer the versions of the frontend and backend to match the SCM tag, so I'm -1 on doing this change.